scripts/version/set-version.ps1

```diff
@@ -9,7 +9,11 @@ This script allows to set the specified version in all required files.
 Param(
     [Parameter(Mandatory = $True, Position = 1)]
     [ValidatePattern("^\d{1,3}\.\d{1,3}\.\d{1,3}$")]
-    [string]$version
+    [string]$version,
+
+    [Parameter(Mandatory = $False, Position = 1)]
```

Comment:
Fix the [Position argument](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_functions_advanced_parameters?view=powershell-7.4#position-argument) for both parameters by removing it from both parameters ($version and $prereleaseSuffix). They are both wrong on all levels.